src/components/ResultsView.tsx

```diff
@@ -546,6 +613,27 @@ const ResultsView: ResultsViewComponent = ({
                       setIsEditViews(true);
                     }}
                   />
+                  <MenuItem
+                    icon={showInterface ? "th-disconnect" : "th"}
+                    text={showInterface ? "Hide Interface" : "Show Interface"}
+                    onClick={() => {
+                      const resultNode = getSubTree({
+                        key: "results",
+                        parentUid,
+                      });
+                      const layoutNode = getSubTree({
+                        key: "layout",
+                        parentUid: resultNode.uid,
+                      });
```

Comment:
IIUC, this is only being used in table correct? If so, it could be treated similarly to how we are treating `Row Style`. If not, meaning we foresee using it in other layouts, then let's remove it from the layout tree and add it to the results tree

Reply:
It currently applies to all layouts.

https://github.com/dvargas92495/roamjs-query-builder/assets/3792666/4709bf83-3859-4a4b-aedf-0e3b5983a2c6